optaplanner-examples/pom.xml

```diff
@@ -42,6 +42,20 @@
   </repositories>
 
   <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <configuration>
+          <ignoredUsedUndeclaredDependencies>
+            <dependency>org.optaplanner:optaplanner-core:jar</dependency>
+          </ignoredUsedUndeclaredDependencies>
+          <ignoredUnusedDeclaredDependencies>
+            <dependency>org.optaplanner:optaplanner-solver:jar</dependency>
+          </ignoredUnusedDeclaredDependencies>
+        </configuration>
+      </plugin>
+    </plugins>
```

Comment:
This is my main pain point with the "optaplanner-solver" empty jar approach.
Everyone who switches from optaplanner-core to optaplanner-solver could run into this.